fetch/cache_test.go

```diff
@@ -15,11 +15,12 @@ import (
 )
 
 // getCachedEntry is a thread-safe cache get helper.
-func getCachedEntry(cache *HashPeersCache, hash types.Hash32) (HashPeers, bool) {
+func getCachedEntry(cache *HashPeersCache, hash types.Hash32) HashPeers {
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
 
-	return cache.get(hash)
+	hp, _ := cache.get(hash)
+	return hp
```

Comment:
I feel like either here or where this helper is called an assertion for the boolean is missing (I believe it should always be true in the current tests).